tests/llama_stack/conftest.py

```diff
@@ -61,6 +64,7 @@ def llama_stack_server_config(
                 "requests": {"cpu": "250m", "memory": "500Mi"},
                 "limits": {"cpu": "2", "memory": "12Gi"},
             },
+            "command": ["/bin/sh", "-c", "llama stack run /etc/llama-stack/run.yaml"],  # Necessary for v2.24
```

Comment:
Main points to 2.25 now. Is this still needed?

Reply:
I'll investigate thanks for letting me know. I might not need the config map too if that is the case